src/components/modals/AddAccounts/steps/03-step-import.js

```diff
@@ -89,7 +89,7 @@ const SectionAccounts = ({ defaultSelected, ...rest }: *) => {
     if (defaultSelected && rest.onSelectAll) {
       rest.onSelectAll(rest.accounts)
     }
-  }, [])
+  }, [defaultSelected, rest])
```

Comment:
this one will be problematic I think. rest always changes, right? this is meant to be exec once on mount.

Reply:
@gre hmmm but if we actually use `onSelectAll` and `accounts` in that component should we not declare them as proper props ? I think `rest` should be passed "as is" and not be interacted with

Reply:
This is OK anyway, as non providing an input arg solve the problem. The eslint hook rule doesn't like empty arrays